src/redis/Redis.cpp

```diff
@@ -423,20 +423,18 @@ namespace redis {
      *  ------ Locking ------
      */
 
-    long Redis::acquireLock(const std::string &key, int expirySeconds) {
+    std::optional<long> Redis::acquireLock(const std::string &key, int expirySeconds) {
         // Implementation of single host redlock algorithm
         // https://redis.io/topics/distlock
-        unsigned int lockId = util::generateGid();
+        long lockId = util::generateGid();
```

Comment:
See point re. changing return value of `acquireLock` to `uint32_t` which this returns.